src/FixedDataTable.js

```diff
@@ -312,6 +312,14 @@ class FixedDataTable extends React.Component {
      */
     stopReactWheelPropagation: PropTypes.bool,
 
+    /**
+     * If enabled scroll events will never be default handled.
+     * If disabled/unspecified, scroll events will be default handled if the scroll
+     * doesn't lead to a change in scroll offsets, which is preferable if you like
+     * the page/container to scroll up when the table is already scrolled up max.
+     */
+    stopScrollDefaultHandling: PropTypes.bool,
```

Comment:
Is the prop name and description fine?

Reply:
Could you change the term "default handled" to "bubbled to the browser default handler"?
I'd also change "disabled/unspecified" to "disabled (default when unspecified)"

Otherwise looks good.